tests/sources/test_header_version.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+# Copyright 1996-2019 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test header version."""
+import unittest
+import os
+import fnmatch
+
+ignoredProtos = [
+    "projects/robots/mobsya/thymio/controllers/thymio2_aseba/aseba/clients/studio/plugins/ThymioVPL/UsageProfile.proto",
+    "projects/samples/tutorials/protos/FourWheelsRobot.proto"
+]
+
+
+class TestIcons(unittest.TestCase):
```

Comment:
it seems that the test class name and description have not been updated after copy/paste